examples/06_manipulating_images/plot_threshold_image.py

```diff
@@ -0,0 +1,175 @@
+
+from nilearn import datasets, plotting
+
+cmap = "RdBu_r"
+cut_coords = [-26, -33, 59]
+
+image = datasets.load_sample_motor_activation_image()
+
+plotting.plot_stat_map(
+    image,
+    colorbar=True,
+    cmap=cmap,
+    title="image without threshold",
+    cut_coords=cut_coords,
+)
+
+# %%
+# Image thresholded at 2 when two_sided=True
+# ------------------------------------------
+#
+# Now we will use ``threshold=2`` together with ``two_sided=True`` to
+# threshold the image. When ``two_sided=True``, we can only use positive
+# values for ``threshold``.
+#
+# This will set all image values between -2 and 2 to 0.
+
+import matplotlib.pyplot as plt
+
+from nilearn.image import threshold_img
+
+thresholded_img = threshold_img(
+    image,
+    threshold=2,
+    cluster_threshold=0,
+    two_sided=True,
+    copy=True,
+)
+
+
+fig, axes = plt.subplots(
+    1,
+    2,
+    figsize=(17, 5),
+)
+
+plotting.plot_stat_map(
+    image,
+    colorbar=True,
+    cmap=cmap,
+    title="image without threshold",
+    axes=axes[0],
+    cut_coords=cut_coords,
+)
+
+plotting.plot_stat_map(
+    thresholded_img,
+    colorbar=True,
+    cmap=cmap,
+    title="image thresholded at 2 with two_sided=True",
+    cut_coords=cut_coords,
+    axes=axes[1],
+)
+
+# %%
+# Image thresholded at 2 when two_sided=False
+# -------------------------------------------
+#
+# Now we will use ``threshold=2`` together with ``two_sided=False`` to
+# see the effect.
+#
+# This will set all image values below 2 to 0.
+
+import matplotlib.pyplot as plt
+
+from nilearn.image import threshold_img
+
+thresholded_img = threshold_img(
+    image,
+    threshold=2,
+    cluster_threshold=0,
+    two_sided=False,
+    copy=True,
+)
+
+
+fig, axes = plt.subplots(
+    1,
+    2,
+    figsize=(17, 5),
+)
+
+plotting.plot_stat_map(
+    image,
+    colorbar=True,
+    cmap=cmap,
+    title="image without threshold",
+    axes=axes[0],
+    cut_coords=cut_coords,
+)
+
+plotting.plot_stat_map(
+    thresholded_img,
+    colorbar=True,
+    cmap=cmap,
+    title="image thresholded at 2 with two_sided=False",
+    cut_coords=cut_coords,
+    axes=axes[1],
+)
+
+# %%
+# Image thresholded at -2 when two_sided=False
+# --------------------------------------------
+#
+# Now we will use ``threshold=-2`` together with ``two_sided=False`` to
+# see the effect.
+#
+# This will set all image values above -2 to 0.
+
+import matplotlib.pyplot as plt
+
+from nilearn.image import threshold_img
+
+thresholded_img = threshold_img(
+    image,
+    threshold=-2,
+    cluster_threshold=0,
+    two_sided=False,
+    copy=True,
+)
+
+
+fig, axes = plt.subplots(
+    1,
+    2,
+    figsize=(17, 5),
+)
+
+plotting.plot_stat_map(
+    image,
+    colorbar=True,
+    cmap=cmap,
+    title="image without threshold",
+    axes=axes[0],
+    cut_coords=cut_coords,
+)
+
+plotting.plot_stat_map(
+    thresholded_img,
+    colorbar=True,
+    cmap=cmap,
```

Comment:
We may have to tweak the cmap here to make it more obvious that we are "hiding" part of the data.

Reply:
Is it ok to make it as below for a negative one sided threshold? If you agree so, I'll change positive one sided one accordingly.

![colormap](https://github.com/user-attachments/assets/a0cb8140-7488-47cd-9ae6-4a4fb74a377e)

Reply:
Yes that's exactly what I had in mind
Thanks!

Reply:
LGTM